packages/dds/tree/src/test/rebaserAxiomaticTests.ts

```diff
@@ -380,6 +654,196 @@ function verifyComposeAssociativity<TChangeset>(
 	assertDeepEqual(rightPartialCompositions.at(-1), singlyComposed);
 }
 
+function verifyComposeWithInverseEqualsEmpty<TChangeset>(
+	edits: TaggedChange<TChangeset>[],
+	fieldRebaser: BoundFieldChangeRebaser<TChangeset>,
+) {
+	const metadata = defaultRevisionMetadataFromChanges(edits);
+
+	for (const edit of edits) {
+		const change = tagChange(edit.change, edit.revision);
+		const changeset = fieldRebaser.compose(
+			change,
+			tagChange(fieldRebaser.invert(change), change.revision),
+			metadata,
+		);
+		const actualChange = makeAnonChange(changeset);
+		assert(fieldRebaser.isEmpty !== undefined);
+		assert(fieldRebaser.isEmpty(actualChange.change));
+	}
+}
+
+function verifyComposeWithEmptyIsNoOp<TChangeset>(
+	edits: TaggedChange<TChangeset>[],
+	fieldRebaser: BoundFieldChangeRebaser<TChangeset>,
+) {
+	const metadata = defaultRevisionMetadataFromChanges(edits);
+	const assertDeepEqual = getDefaultedEqualityAssert(fieldRebaser);
+	for (const edit of edits) {
+		const emptyChange = tagChange(
+			fieldRebaser.createEmpty(),
+			"test" as unknown as SessionSpaceCompressedId,
+		);
+		const changeset = fieldRebaser.compose(edit, emptyChange, metadata);
```

Comment:
The name of this test suggests that it should also check that `compose(emptyChange, edit) === edit`.

Reply:
Added additional check